server-spi/src/main/java/org/keycloak/models/PasswordPolicy.java

```diff
@@ -97,6 +99,14 @@ public int getExpiredPasswords() {
         }
     }
 
+    public int getPasswordAgeInDays() {
+        if(policyConfig.containsKey(PASSWORD_AGE)) {
```

Comment:
```suggestion
        if (policyConfig.containsKey(PASSWORD_AGE)) {
```

Reply:
Updated branch with suggested changes. I also increased the time offset in tests, it was set to a couple of seconds, but now it is a couple of minutes. It should help with test flakiness in case of longer runs.